static/scripts/util.js

```diff
@@ -52,9 +52,57 @@ function addBadge(selectElement, containerSelector) {
     }
 }
 
+function changeBadgeColorCheckbox(checkboxValue) {
+    const badge = $('span.badge', checkboxValue.parent());
+    if (checkboxValue.is(':checked')) {
+        badge.removeClass('bg-secondary').addClass('bg-primary');
+    } else {
+        badge.removeClass('bg-primary').addClass('bg-secondary');
+    }
+}
+
+function filter(page) {
+    let activeOrganizations = $('.badge-checkbox:checked').map(function () {
+        return $(this).data('id');
+    }).get();
+
+    let items;
+    if (page === "course") {
+        items = $('.course-item')
+    } else if (page === "user") {
+        items = $('.user-item')
+    }
+
+    if (activeOrganizations.length > 0) {
+        items.each(function () {
+            let organizations;
+            let showItem;
+
+            if (page === "course") {
+                organizations = $(this).data('organizations').split(',');
```

Comment:
```suggestion
                organizations = $(this).data('organizations').toString().split(',');
```

If you have one org only, the numerical id will be cast automatically to an Integer by Javascript...